notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    if os.getenv("IS_TESTING"):
</pre><p>BUCKET_URI is never defined. Please remove.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>

Reply:
Good catch, this should be `BUCKET_NAME`.